sonaranalyzer-dotnet/src/Tests/SonarAnalyzer.UnitTest/TestCases/UriPropertiesShouldNotBeStrings.cs

```diff
@@ -0,0 +1,56 @@
+using System;
+
+namespace Tests.Diagnostics
```

Comment:
Also, you may want to separate Compliant and Noncompliant because it is pretty packed and so hard to read.